Makefile.include

```diff
@@ -290,6 +290,13 @@ term: $(filter flash, $(MAKECMDGOALS))
 doc:
 	make -BC $(RIOTBASE) doc
 
+btest:
```

Comment:
Should this be really go into the general Makefile or stay something board specific?